styles.css

```
@@ -0,0 +1,260 @@
+    border-radius: 4px;
+    background-color: #FFFFFF;
+    margin-top: 76px;
+    margin-left: auto;
+    margin-right: auto;
+    position: relative;
+    
+}
+
+
+.main-title{
+
+    width: 382px;
+    height: 24px;
+    position: absolute;
+    top: 52px;
+    left: 16px;
+
+    font-weight: 700;
+    font-size: 24px;
+    line-height: 24px;
+    letter-spacing: 0.15px;
+    display: flex;
+    justify-content: center;
+    
+    
+
+}
+
+.txt1{
+    width: 310px;
+    height: 24px;
+
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    letter-spacing: 0.5px;
+    position: absolute;
+    top: 104px;
+    left: 16px;
+
+}
+
+.input-txt{
+
+    width: 310px;
+    height: 28px;
+    border-radius: 4px;
+    border: 1px solid #B4B4B4;
+    padding-top: 8px;
+    padding-right: 8px;
+    padding-left: 8px;
+    position: absolute;
+    top: 132px;
+    left: 16px;
+
+}
+
+.buy-button{
+
+    width: 53px;
+    height: 36px;
+    position: relative;
+    top: 132px;
+    left: 355px;
+    border-radius: 4px;
+    border: none;
+    padding-top: 6px;
+    padding-left: 8px;
+    padding-bottom: 6px;
+    padding-right: 6px;
+    background-color: #4E5BA6;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    letter-spacing: 1.25px;
+    color: #FFFFFF;
+    
+}
+
+.txt2{
+    width: 310px;
+    height: 24px;
+    position: absolute;
+    top: 196px;
+    left: 16px;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    letter-spacing: 0.5px;
+
+}
+
+.ticket-list{
+    width: 238px;
+    height: 276px;
+    position: absolute;
+    top: 228px;
+    left: 16px;
+}
+
+.ticket-item{
+
+    width: 238px;
+    height: 36px;
+    display: flex;
+    align-items: center;
+}
+
+.ticket-icon{
+    font-weight: 600;
+    font-size: 34px;
+    line-height: 36px;
+}
+
+.ticket-numbers{
+
+    width: 196px;
+    height: 24px;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    letter-spacing: 0.5px;
+    text-align: left;
+}
+
+.txt3{
+
+    width: 400px;
+    height: 24px;
+    position: absolute;
+    top: 528px;
+    left: 16px;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    letter-spacing: 0.5px;
+}
+
+.txt4{
+
+    width: 70px;
+    height: 24px;
+    position: absolute;
+    top: 564px;
+    left: 16px;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    letter-spacing: 0.5px;
+}
+
+.txt5{
+
+    width: 85px;
+    height: 24px;
+    position: absolute;
+    top: 564px;
+    left: 322px;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    letter-spacing: 0.5px;
+
+}
+
+.numbers-input{
+
+width: 260px;
+height: 36px;
+position: absolute;
+top: 596px;
+left: 16px;
+border-radius: 4px 0px 0px 0px;
+border: 1px 0px 0px 0px;
+
+}
+
+.lotto-number{
+
+width: 30px;
+height: 36px;
+gap: 0px;
+border-radius: 4px;
+border: 1px;
+border: 1px solid #B4B4B4
+
+}
+
+.bonus-number{
+
+width: 30px;
```

```suggestion
.bonus-number {
  width: 30px;
```

이런 부분들에서도 탭 지켜주시는 게 더 보기 깔끔할 것 같아요~